### PR TITLE
Cleanup unneeded language specific

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs [hunk 1]
```diff
@@ -93,10 +94,11 @@ private void AnalyzeSymbol(INamedTypeSymbol symbol, INamedTypeSymbol attributeTy
             }
         }
 
-        protected abstract bool IsAssignableTo(
+        private static bool IsAssignableTo(
             [NotNullWhen(returnValue: true)] ITypeSymbol? fromSymbol,
             [NotNullWhen(returnValue: true)] ITypeSymbol? toSymbol,
-            Compilation compilation);
+            Compilation compilation)
```
Comment (genlu) [inline src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs:100]: @mavasani Is there a common place for helpers like this? The code is identical in both analyzers

**Review by genlu — APPROVED** [review]

File: src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs (hunk 1 above)
Comment (Evangelink) [reply src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs:100]: [ITypeSymbolExtensions](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/Compiler/Extensions/ITypeSymbolExtensions.cs) looks like a good candidate.

Comment (Youssef1313) [reply src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs:100]: @Evangelink I attempted to move it as an extension method there, but `ClassifyCommonConversion` doesn't seem to be available on public api analyzers, which references this shared project.

image

**Comment by Evangelink** [comment]
@Youssef1313 You have some unused usings.

**Review by Evangelink — APPROVED** [review]

**Pull request merged**